web/src/app/wizard/WizardRouter.js

```diff
@@ -45,8 +45,9 @@ const styles = {
 
 @withWidth()
 @withStyles(styles)
-export default class WizardRouter extends React.PureComponent {
-  state = {
+
+export default function WizardRouter ({classes, width}) {
+ const [state, setState] = useState({
```

Comment:
When we convert from `this.state` on a Class Component to `useState` on a hooks component, we'll want to have a call to `useState` for each main key:value pair.

e.g.
```
const [errorMessage, setErrorMessage] = useState(null)
const [complete, setComplete] = useState(false)
const [redirect, setRedirect] = useState(false)
const [value, setValue] = useState(...) // everything in that value obj
```